src/hotspot/share/prims/jvmtiAgent.hpp

```diff
@@ -43,6 +43,10 @@ class JvmtiAgent : public CHeapObj<mtServiceability> {
   const char* _options;
   void* _os_lib;
   const char* _os_lib_path;
+#ifdef AIX
+  long _inode;
+  long _device;
```

Comment:
How are dev_t and ino_t defined on AIX, I'm wondering if long is okay here.

Reply:
They are defined as __ulong64_t which is unsigned long. So I can change it to unsigned long or even to dev_t and ino_t.